contracts/child/MaticChildERC20.sol

```diff
@@ -0,0 +1,160 @@
+pragma solidity ^0.5.2;
+
+import { ChildToken, ERC20, LibTokenTransferOrder } from "./ChildChain.sol";
+
+
+contract ChildERC20 is ChildToken, ERC20, LibTokenTransferOrder {
+
+  event Deposit(
+    address indexed token,
+    address indexed from,
+    uint256 amount,
+    uint256 input1,
+    uint256 output1
+  );
+
+  event Withdraw(
+    address indexed token,
+    address indexed from,
+    uint256 amount,
+    uint256 input1,
+    uint256 output1
+  );
+
+  event LogTransfer(
+    address indexed token,
+    address indexed from,
+    address indexed to,
+    uint256 amount,
+    uint256 input1,
+    uint256 input2,
+    uint256 output1,
+    uint256 output2
+  );
```

Comment:
Code duplication is the worst. Move these events to another contract and make childErc20 and maticChild inherit from it.

Reply:
true should be covered in childErc20 refactoring.